app/controllers/projects_controller.rb

```diff
@@ -1,10 +1,12 @@
 class ProjectsController < ApplicationController
   def index
-    @featured = Project.featured
+    @featured = Project.includes(logo_attachment: :blob).featured
```

Comment:
For what it's worth, [`ActiveStorage` adds a `with_logo_attachment` scope](https://api.rubyonrails.org/classes/ActiveStorage/Attached/Macros.html#method-i-has_one_attached) to do this for you.

Reply:
Nice!